com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/FrameSettings.cs

```diff
@@ -158,6 +158,12 @@ public enum FrameSettingsField
         /// <summary>When enabled, Cameras using these Frame Settings render water surfaces.</summary>
         [FrameSettingsField(0, autoName: Water, customOrderInGroup: 106, tooltip: "When enabled, Cameras using these Frame Settings render water surfaces.")]
         Water = 99,
+        /// <summary>When enabled, HDRP accounts for asymmetry in the projection matrix.</summary>
```

Comment:
I am a bit worried about this statement as it it not totally true, effect like volumetric fog will fail with (strong) asymetric matrix. But I am not sure what to say instead :P

Reply:
made the comment more specific based on this frame setting usage